adventureGame/AdventureGame/.spyproject/unittest.ini

```diff
@@ -1,6 +1,6 @@
 [unittest]
 framework = pytest
-wdir = C:/Users/mr_ep/OneDrive/Documents/University/Software Development/Assignment 2/adventureGameRepo/AdventureGame/adventureGame\AdventureGame
```

Comment:
Change line to:
wdir = C:/Users/mr_ep/OneDrive/Documents/University/Software Development/Assignment 2/adventureGameRepo/AdventureGame/adventureGame/AdventureGame